dts/msm8974/msm8974-oneplus-bacon.dts

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	qcom,msm-id = <194 0x10000>;
+	qcom,board-id = <0x8 0x0>;
```

Comment:
This looks like `msm8974pro-ac-pm8941-mtp.dts` (8 0 is always standard -mtp from qcom). You'll need the same setup as for FP2 here, with a match property for oneplus-bacon so other devices can be added to the same dts.

Reply:
fixed